packages/api/models/executions.js

```diff
@@ -40,7 +43,7 @@ class Execution extends Manager {
     const arn = getMessageExecutionArn(cumulusMessage);
     if (isNil(arn)) throw new Error('Unable to determine execution ARN from Cumulus message');
 
-    const status = get(cumulusMessage, 'meta.status');
+    const status = getWorkflowStatus(cumulusMessage);
```

Comment:
I'm a bit torn on these abstractions.   On the one hand, not using strings everywhere is a good abstraction, on the other hand, `workflowStatus` is an indirect reference to meta.status, but changing it to getMetaStatus defeats the abstraction....and also we're putting things in 'meta' like 'status' that users could easily accidently overwrite.  ( https://nasa.github.io/cumulus/docs/workflows/cumulus-task-message-flow#docsNav) 

🤔   

This is a sidebar to this PR, as given while I type this I'm realizing it's more about the `meta` namespace than the abstraction itself I don't like.   Leaving here for posterity/to trigger a discussion.

Reply:
OK. I guess I don't totally follow. The only point of `getWorkflowStatus` and other functions in the `message` package is to have trivial helpers for extracting information from the workflow message, which we're now doing in at least two places (Dynamo models and RDS write code) and possibly more in the future. Is the concern the function name?

Reply:
Yeah, sorry, I could have expressed that better.      Reading through it with review blinders on, I'm not sure I  know what 'workflowStatus' is as an entity on a Cumulus message, except via the helper source that tells me it means meta.status.   

(having it in `meta` is in userspace is a separate discussion I'm sure we already had, but `status` could easily be accidently overrode by a user as we don't define anything in meta as reserved) 

tldr:  My concern would be resolved if this sort of method was named getMetaStatus, but following *that* pattern seems like a thin reason to wrap cumulusMessage?.meta?.status in TS as an update, although perhaps justified in vanilla node.    

It'd also be resolved if it were in cumulus_meta and that object were defined better-*er*, but that starts to *really* feel outside scope.

Reply:
I think the reason to wrap `cumulusMessage?.meta?.status` in a helper is just to avoid having that logic duplicated, decreasing the chances (however small) of bugs. Let me know if you disagree though. I'm fine with renaming it to `getMetaStatus`.

Reply:
And to be clear, this is absolutely nit territory.    We should time box more thought on this to, say, 15 minutes.

Reply:
> I'm fine with renaming it to getMetaStatus.

That seems fine.  If it causes annoyance and we actually ever move it from meta, we can put a PR up and discuss there.

Reply:
0f84f40d83d9a2d158d4933177eb43d92b64a113